### PR TITLE
docs: add cross domain section for typescript browser sdk

### PR DESCRIPTION
**Pull request by liuyang1520**
# Amplitude Developer Docs PR
## Description
- docs: add cross domain section for typescript browser sdk

image

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/sdks/typescript-browser/index.md [hunk 1]
```diff
@@ -566,6 +566,22 @@ add(new MyDestinationPlugin('https://custom.domain.com'));
...
+
+You can track anonymous behavior across two different domains. Amplitude identifies anonymous users by their device IDs which must be passed between the domains. For example:
+
+- Site 1: `www.example.com`
+- Site 2: `www.example.org`
+
+Users who start on Site 1 and then navigate to Site 2 must have the device ID generated from Site 1 passed as a parameter to Site 2. Site 2 then needs to initialize the SDK with the device ID.
+ The SDK can parse the URL parameter automatically if `deviceId` is in the URL query parameters.
+
+- From Site 1, grab the device ID from `getDeviceId()`.
```
Comment (caseyamp) [inline docs/data/sdks/typescript-browser/index.md:579]:
```suggestion
1. From Site 1, grab the device ID from `getDeviceId()`.
```

File: docs/data/sdks/typescript-browser/index.md [hunk 2]
```diff
@@ -566,6 +566,22 @@ add(new MyDestinationPlugin('https://custom.domain.com'));
...
+- Pass the device ID to Site 2 via a URL parameter when the user navigates. (for example: `www.example.com?deviceId=device_id_from_site_1`)
+- Initialize the Amplitude SDK on Site 2 with `init('API_KEY', null)`.
```
Comment (caseyamp) [inline docs/data/sdks/typescript-browser/index.md:581]:
```suggestion
3. Initialize the Amplitude SDK on Site 2 with `init('API_KEY', null)`.
```

Comment (liuyang1520) [reply docs/data/sdks/typescript-browser/index.md]: Thank you for reviewing! Updated! I also update the JS one to number list to be aligned.

**Review by yuhao900914 — APPROVED** [review]
The content looks accurate to me. 
Please also get Casey's approval.

**Review by caseyamp — APPROVED** [review]

**Pull request merged**